README.rst

```diff
@@ -149,6 +149,23 @@ Do not forget to clone astroid_ and install the last version::
     cd astroid
     python3 -m pip install -e .
 
+Share your code style
+-------
```

Comment:
```suggestion
Show your usage
-----------------
```

Linting is not really code style, although `Show your usage` isn't perfect either.